src/main/java/com/minecolonies/colony/buildings/AbstractBuilding.java

```diff
@@ -340,24 +360,24 @@ public int getBuildingLevel()
     /**
      * Sets the current level of the building
      *
-     * @param level     Level of the building
+     * @param level
+     *            Level of the building
      */
     public void setBuildingLevel(int level)
     {
-        if(level > getMaxBuildingLevel())
-        {
-            return;
-        }
+        if (level > getMaxBuildingLevel()) { return; }
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS00122)